src/main/kotlin/net/irisshaders/lilybot/extensions/events/MessageDelete.kt

```diff
@@ -10,27 +10,35 @@ import dev.kord.core.event.message.MessageDeleteEvent
 import kotlinx.datetime.Clock
 import net.irisshaders.lilybot.utils.DatabaseHelper
 
+/**
+ * The class for logging deletion of messages to the guild action log.
+ *
+ * @since 2.0
+ */
 class MessageDelete : Extension() {
 	override val name = "message-delete"
 
 	override suspend fun setup() {
 		/**
 		 * Logs deleted messages in a guild to the message log channel designated in the config for that guild
 		 * @author NoComment1105
+		 * @since 2.0
 		 */
 		event<MessageDeleteEvent> {
 			// Don't try to create if the message is in DMs
 			check { anyGuild() }
+
 			action {
-				if (event.message?.author?.isBot == true || event.message?.author?.id == kord.selfId) return@action
+				// Fail if the user is a bot
+				if (event.message?.author?.isBot == true) return@action
 
 				// Try to get the message logs channel, return@action if null
```

Comment:
This comment is no longer what actually happens. And also doesn't exist other places so can be removed.